index.bs

```diff
@@ -1665,9 +1662,8 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
-1. If the <code>|options|.{{CredentialCreationOptions/signal}}</code> is present and its
-    [=AbortSignal/aborted flag=] is set to [TRUE], return a {{DOMException}} whose name is "{{AbortError}}"
-    and terminate this algorithm.
+1. If the <code>|options|.{{CredentialCreationOptions/signal}}</code> is present and [=AbortSignal/aborted=], throw
+    the <code>|options|.{{CredentialCreationOptions/signal}}</code>'s [=AbortSignal/abort reason=].
```

Comment:
Regarding the second part, I guess since we have the link to the DOM spec using `[=AbortSignal/abort reason=]`, it might be intuitive, but I slightly prefer explicitly specifying where the abort reason comes from for clarity and so that it matches other specs' language. I've removed the first "the" though.